start_redis_task.py

```diff
@@ -1,9 +1,5 @@
-activate_this = '/var/www/mybuzzn-backend.buzzn.net/mybuzznbackend/env/bin/activate_this.py'
-with open(activate_this) as file_:
-    exec(file_.read(), dict(__file__=activate_this))
-
 import setup_environment
-    
 import util.task
+
 util.task.run()
```

Comment:
Could you explain why you made these changes, please?

Reply:
The setup is done in the setup_envornment. The service will execute this script in the virtualenv, so the script does not create the virtualenv anymore.

Reply:
The service will start the script in a virtual env, so there is no need for the script to setup the virtualenv.